bot.py

```diff
@@ -84,11 +84,34 @@ async def daily_reminder(ctx, time):
 @client.command(aliases=["start-pomodoro"])
 async def pomodoro(ctx, time):
     # Set a timer for 25 minutes
+    time = 1500
     # Notify the user after 25 minutes
+    message = await ctx.send("Your countdown has been started for 25 minutes")
+    while True:
+        mins, sec = divmod(time, 60)
+        time -= 1
+        if time == 0:
+            await message.edit(content=("Time remaining {:02d}:{:02d}".format(0, 0)))
+            await ctx.send(ctx.message.author.mention + " your countdown has ended!\nBreak starts.")
+            break
+
+        await message.edit(content=("Time remaining {:02d}:{:02d}".format(mins, sec)))
```

Comment:
Please remove second-wise updates as it would result in too many requests to the server, which it might not be able to handle.

Reply:
Okay on it. I actually added it by referring to the start_timer function in bot.py file as it has second-wise update.